src/axiosDefaultConfig.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+export const axiosDefaultConfig = axios.create({
+  baseURL: import.meta.env.VITE_API_URL,
+  headers: {
+    Authorization: import.meta.env.VITE_API_KEY,
```

Comment:
I think that TS error here occurs because we are missing some types. Probably, we need add `@types/node`